ui_utils/interaction_manager/demo/Gamepad/gamepad-buttons-and-axes.html

```diff
@@ -0,0 +1,170 @@
+                        fill="#A9A8A9"
+                        d="M499.7,65c-48.2-13.6-98.2-19.1-148.1-20.3c-50.5-1.2-101.1,1.9-151.5,5.4c-28.2,2-56.5,4.4-83.8,12.4    c0.8,0.2,1.6,0.4,2.4,0.6c-0.5-0.5-1-1-1.4-1.5c-1.2-1.2-3-0.8-3.9,0.5C92.1,92.9,82.7,129.8,75.3,166c-4.2,20.6-8.1,41.3-10,62.2    c-1.9,20.8-1.7,41.8,1,62.5c2.1,16.6,5.2,33.8,13.1,48.8c3.6,6.7,8.2,13,14.2,17.7c2.6,2.1,6,4.9,9.3,5.5c1.5,0.2,3,0.1,4.5,0.2    c2.3,0.1,4.7,0.6,6.9,1.1c8.4,1.7,17.2,3.5,24.5-2.2c6.5-5,9.9-13.2,12.8-20.6c5.7-14.9,8.2-30.7,13-45.8c2.3-7.3,5.1-14.5,9.2-21    c4.2-6.7,9.9-13.4,17.2-16.6c9.3-4.1,18.5,1.8,28,1.4c8.9-0.3,17.1-4.4,25.9-5.2c2.1-0.2,4.2-0.2,6.2,0.2c1.9,0.3,2.8,0.3,4.2-1.1    c-0.8,0.2-1.6,0.4-2.4,0.6c23.9,4,48.3,5.4,72.5,4c5.9-0.3,11.8-0.9,17.7-1.5c5.9-0.7,11.9-1.4,17.9-1.2c11.6,0.4,22.4,6,34.2,5.2    c5.9-0.4,11.7-1.7,17.7-1.4c4.7,0.3,9.3,1.9,13.4,4.2c8.6,4.9,14.9,12.7,19.1,21.5c9.8,20.2,9.7,43.6,18.4,64.2    c2.1,5,4.6,10.6,9.1,14.1c4.6,3.6,10.5,3.6,16.1,3.5c11-0.2,22.5-1,31.5-8.1c-0.6,0.2-1.2,0.5-1.8,0.7c3.8-0.1,6.9-5,9.1-7.8    c2.7-3.4,5.1-7,7.2-10.8c4.4-7.9,7.4-16.6,9.5-25.4c4.6-18.8,5.5-38.5,5.6-57.8c0.2-42.2-6.2-84.5-18.9-124.8    c-7.1-22.6-16.2-44.6-27.2-65.7c-1.5-2.9-5.8-0.3-4.3,2.5c18.7,35.9,31.8,74.7,39,114.6c3.6,20.1,5.7,40.5,6.2,60.9    c0.5,19.1,0.1,38.6-3,57.5c-3.2,19.3-9.7,38.6-24.2,52.4c0.6-0.2,1.2-0.5,1.8-0.7c-2.2,0.1-3.4,1.4-5.3,2.5    c-2.2,1.3-4.5,2.2-7,2.9c-4.9,1.4-9.9,1.7-14.9,1.9c-5,0.2-11.1,1-15.5-1.7c-4.6-2.7-7-9-8.9-13.7c-8.4-20.7-8.4-44-18.4-64.2    c-4.3-8.7-10.6-16.5-18.9-21.8c-4-2.6-8.5-4.5-13.2-5.4c-5.3-1-10.6-0.5-15.9,0.1c-6.2,0.8-11.9,1-18-0.3    c-5.6-1.2-11.1-3-16.8-3.7c-12.2-1.7-24.6,1.1-36.8,1.9c-13.7,1-27.5,1.1-41.2,0.4c-6.7-0.4-13.4-1-20.1-1.8    c-3.5-0.4-7-0.9-10.5-1.4c-2.7-0.4-5.1-1.5-7.2,0.6c0.8-0.2,1.6-0.4,2.4-0.6c-8.6-1.9-16.9,0.9-25.2,3c-4.1,1-8.3,1.9-12.6,1.5    c-4.8-0.4-9.4-2.1-14.2-2.7c-16.6-2.1-29.1,13.4-35.7,26.7c-6.7,13.7-9.7,28.8-13.5,43.5c-2,7.4-4.1,14.9-7.2,21.9    c-3.1,7-7.3,15.7-15.7,17.1c-4.4,0.7-8.9-0.4-13.2-1.3c-4.2-0.9-8.2-1.2-12.4-1.4c-0.2,0-0.6,0-0.8-0.1c-0.4-0.1-0.4-0.3,0.1,0    c-0.8-0.4-1.6-0.9-2.4-1.3c-1.5-0.9-3-2-4.4-3.1c-2.8-2.3-5.4-5-7.6-7.9c-9.4-12.4-13.3-28.1-16-43.2c-3.5-20-4.6-40.4-3.6-60.7    c1-20.4,4.2-40.7,8.1-60.8c7-35.8,14.7-72.8,33.2-104.7c2.2-3.8,4.6-7.5,7.1-11.2c-1.3,0.2-2.6,0.3-3.9,0.5c0.5,0.5,1,1,1.4,1.5    c0.6,0.6,1.6,0.9,2.4,0.6c24.1-7.1,49.2-9.7,74.2-11.6c25-1.9,50-3.5,75-4.7c49.6-2.4,99.4-2.7,148.8,2.9    c27.9,3.2,55.6,8.3,82.7,15.9C501.5,70.7,502.8,65.9,499.7,65L499.7,65z"
+                    />
+                </g>
+            </g>
+        </svg>
+
+        <script src="../../dist/Gamepad.js"></script>
+        <script>
+            Gamepad.enabled = true;
+
+            const buttons = [];
+
+            for (let i = 0; i < 17; i++) {
+                buttons.push(document.getElementById(i));
+            }
+
+            buttons.forEach((button, index) => {
+                Gamepad.on({
+                    actions: [index],

```

Comment:
This a fallback in case developers want to unify between platforms. In reality the Gamepad API provides an array of buttons and you are able to provide the button index if you don't want to be platform specific. For example you can do 'playstation.x' or 'xbox.b' which will both correspond to the button at the 0 index in the array. If at some point you don't want to use these, you can just write 0 and it will still work.

PS. Maybe a good idea would be to allow users to create their own aliases 🤔

Reply:
Aliases seems reasonable